doc/tutorial/pairwise_alignment/index.md

```diff
@@ -243,6 +243,11 @@ a simple scoring scheme with mismatch `-2` and match `4`.
 
  -  TTACGTACGGACTAGCTACAACATTACGGACTAC
  -  GGACGACATGACGTACGACTTTACGTACGACTAGC
+
+\hint
+An alignment is called overlap alignment if it allows free end-gaps at each sequence end. In this configuration
```

Comment:
```suggestion
An alignment is called overlap alignment if it allows free end-gaps at each sequence end. With this configuration,
```

Not too sure about the comma but the `With`

Reply:
There shouldn't be a comma because `With this configuration` is not a whole sentence.